stylesheets/_structure.grid.scss

```diff
@@ -1,36 +1,72 @@
-@import '../libs/susy/sass/susy';
+$columns:       12;
+$gutter_pc:     1;
 
-// Susy grid settings
-$susy: (
-    columns: 12,
-    container: 100%,
-    gutters: .25,
-);
+@mixin column($first: false) {
+    background-clip: padding-box !important;
+    box-sizing: border-box;
+    display: block;
+    float: left;
+    padding: 0;
+    @if $first {
+        margin-left: 0;
+    }
+    @else {
+        @include respond-min($screen-phone) {
+            margin-left: 0;
+        }
 
-.grid-container {
-    @include container;
+        @include respond-min($screen-tablet) {
+            margin-left: percentage($gutter_pc / 100);
+        }
+    }
 }
 
-@for $i from 1 through 12 {
-    .grid-col--#{$i} {
+@mixin span($num, $force: false, $first: false) {
+    $one_col: (100% - ($gutter_pc * ($columns - 1))) / $columns;
+    @include column($first);
+
+    @if $force == false {
         @include respond-min($screen-phone) {
-            @include span($i);
+            width: 100%;
         }
     }
+    @else {
+        width: ($one_col * ($num)) + ($gutter_pc * ($num - 1));
```

Comment:
Could `($one_col * ($num)) + ($gutter_pc * ($num - 1))` be hoisted into a variable at the start of the block ☝️, it could make this mixin easier to reason about if we reduce those duplicate calculations below maybe?